### PR TITLE
Improve logging for signer vulnerability check

### PR DESCRIPTION
**Pull request by ooq**
- add info logging for vulnz wait time
- add info logging for default severity setting
- better formatting for violations

### CODE REVIEW

File: pkg/kritis/crd/vulnzsigningpolicy/vulnzsigningpolicy.go [hunk 1]
```diff
@@ -45,11 +46,13 @@ func ValidateVulnzSigningPolicy(vsp v1beta1.VulnzSigningPolicy, image string, vu
 
 	maxSev := vsp.Spec.ImageVulnerabilityRequirements.MaximumFixableSeverity
 	if maxSev == "" {
+		glog.Info("maximumFixableSeverity is unset, default to Critical.")
 		maxSev = constants.Critical
```
Comment (nenaddedic) [inline pkg/kritis/crd/vulnzsigningpolicy/vulnzsigningpolicy.go:50]: Isn't this equivalent to AllowAll?

File: pkg/kritis/metadata/containeranalysis/containeranalysis.go [hunk 2]
```diff
@@ -345,6 +347,8 @@ func (c Client) WaitForVulnzAnalysis(containerImage string, timeout time.Duratio
 			}
 			switch updated.GetDiscovered().GetDiscovered().GetAnalysisStatus() {
 			case discovery.Discovered_FINISHED_SUCCESS:
+				elapsed := time.Since(start)
+				glog.Infof("Find vulnerabilities after %s", elapsed)
```
Comment (nenaddedic) [inline pkg/kritis/metadata/containeranalysis/containeranalysis.go:351]: Find -> Found
(if that's what you meant)

File: pkg/kritis/crd/vulnzsigningpolicy/vulnzsigningpolicy.go (hunk 1 above)
Comment (ooq) [reply pkg/kritis/crd/vulnzsigningpolicy/vulnzsigningpolicy.go:50]: Essentially yes with the tiny difference that `AllowAll` will also pass unknown severity levels. The values are inherited from image security policy, and I was also thinking if they are redundant.

File: pkg/kritis/metadata/containeranalysis/containeranalysis.go (hunk 2 above)
Comment (ooq) [reply pkg/kritis/metadata/containeranalysis/containeranalysis.go:351]: ack.

**Pull request merged**